test/OrchardCore.Commerce.Tests.UI/Tests/BasicTests/SecurityScanningTests.cs

```diff
@@ -41,19 +40,14 @@ public Task FullSecurityScanShouldPass() =>
                         @".*/\?.*products\..*");
                 },
                 maxActiveScanDurationInMinutes: 5,
-                maxRuleDurationInMinutes: 1),
-            changeConfiguration: configuration => configuration.AssertAppLogsAsync = async webApplicationInstance =>
-            {
-                var logsWithoutUnwantedExceptionMessages = (await webApplicationInstance.GetLogOutputAsync())
-                    .SplitByNewLines()
-                    .Where(message =>
-                        !message.ContainsOrdinalIgnoreCase("System.IO.DirectoryNotFoundException: Could not find a part of the path") &&
-                        !message.ContainsOrdinalIgnoreCase(
-                            "System.IO.IOException: The filename, directory name, or volume label syntax is incorrect") &&
-                        !message.ContainsOrdinalIgnoreCase("System.InvalidOperationException: This action intentionally causes an exception!"));
-
-                logsWithoutUnwantedExceptionMessages.ShouldNotContain(item => item.Contains("|ERROR|"));
-            });
+                maxRuleDurationInMinutes: 1,
+                additionalPermittedErrorLinePatterns:
+                [
+                    "System.IO.DirectoryNotFoundException: Could not find a part of the path",
+                    "System.IO.IOException: The filename, directory name, or volume label syntax is incorrect",
+                    "System.InvalidOperationException: This action intentionally causes an exception!",
+                    nameof(ExceptionContextLoggingMiddleware),
```

Comment:
Doesn't this cause _every_ error to be ignored?

Reply:
At first glance that still seems too far reaching to me.

Reply:
`ExceptionContextLoggingMiddleware` detects when a request has thrown an exception and logs additional information about its HTTP context for easier diagnostics. It only uses `logger.LogError` for visibility: it's always next to a real error log anyway so, this simplifies the logger config.

If the real error is already suppressed, then not suppressing `ExceptionContextLoggingMiddleware` will cause a false error detection in `AssertAppLogsAsync`. If the real error is not suppressed, then we'll have a detection anyways and it doesn't matter. So logically this log should always be suppressed.

Reply:
Oh, OK, so it's just additional logging, but if there's a real error we shouldn't ignore, that won't have `ExceptionContextLoggingMiddleware` in its text.
I see you've got a UITT PR with this change, but I'll just add it directly in the UI test in my PR now, while I'm working on the rest of the DotNest upgrade.